static/samples/kind-cluster.yaml

```diff
@@ -1,5 +1,7 @@
 apiVersion: kind.x-k8s.io/v1alpha4
 kind: Cluster
+networking:
+  ipFamily: ipv6
```

Comment:
IPV6 is a requirement?

Reply:
No, didn't mean to commit this. I was testing something completely different :sweat_smile:

Reply:
Removed.